docs/csharp/whats-new/csharp-7.md

```diff
@@ -521,8 +520,8 @@ feature:
 [!code-csharp[UsingValueTask](../../../samples/snippets/csharp/new-in-7/AsyncWork.cs#30_UsingValueTask "Using ValueTask")]
 
 > [!NOTE]
-> You need to add the pre-release NuGet package `System.Threading.Tasks.Extensions`
-> in order to use `ValueTask` in Visual Studio 15 Preview 5.
+> You need to add the NuGet package `System.Threading.Tasks.Extensions`
```

Comment:
I don't see the point. This whole article uses newlines to keep the text lines at about the same length. And in Markdown, for a newline to matter, there either have to be two of them (in other words, an empty line) or the preceding line has to end with two spaces. Neither is the case here, so this newline shouldn't make a difference.

Also, this way, the diff is cleaner.